en/websites_privacy_notice.md

```diff
@@ -1,26 +1,27 @@
 # Websites, Communications & Cookies Privacy Notice
 
-March 8, 2023
-{: datetime="2023-03-08" }
+November 1, 2023
+{: datetime="2023-11-01" }
 
-We care about your privacy. When Mozilla (that's us) collects information about you, our [Mozilla Privacy Policy](https://www.mozilla.org/privacy/) describes how we handle that information.
+We care about your privacy. When Mozilla (that’s us) collects information about you, our [Mozilla Privacy Policy](https://www.mozilla.org/privacy/) describes how we handle that information.
 
 This privacy notice applies to Mozilla operated websites and mobile apps, which include the domains mozillians.org, mozilla.org, and firefox.com, among others. This includes, for example, foundation.mozilla.org, bugzilla.mozilla.org, reps.mozilla.org, careers.mozilla.org, developer.mozilla.org, support.mozilla.org, addons.mozilla.org, and wiki.mozilla.org.
```

Comment:
```suggestion
This privacy notice applies to Mozilla operated websites and mobile apps, which include the domains mozilla.org, and firefox.com, among others. This includes, for example, foundation.mozilla.org, bugzilla.mozilla.org, community.mozilla.org, careers.mozilla.org, people.mozilla.org, developer.mozilla.org, support.mozilla.org, addons.mozilla.org, and wiki.mozilla.org.
```

Maybe not a blocker, but:
* `mozillians.org` was shut down a while ago, it redirects to `people.mozilla.org`. Should we add that to the second list?
* `reps.mozilla.org` (shut down as of last month) redirects to `community.mozilla.org`, which is not listed here.